### DISCUSSION TITLE
阿里云盘视频格式.mkv支持

### DISCUSSION THREAD

**Discussion by tingwensuojian** [AlistGo/alist#310] [Q&A] [answered]
.mkv格式可以播放但没有声音，望开发大大更新😭

**Comment by xhofe** [comment, marked as answer]
- #227 
- https://alist-doc.nn.ci/docs/common-question#%E8%A7%86%E9%A2%91%E6%92%AD%E6%94%BE%E4%B8%8D%E4%BA%86
